<launch>

  <node name="fsm_lidom_node" pkg="fsm_lidom_ros" type="fsm_node" output="screen">
    <param name="pkg_name"   type="string" value="FSM_LIDOM" />
    <rosparam command="load" file="$(find fsm_lidom_ros)/config/params.yaml" />
  </node>

</launch>
